docs/docs/resources/variables/config_env_vars.md

```diff
@@ -1,11 +1,11 @@
-|KPOPS_SCHEMA_REGISTRY_URL|             |False   |Address of the Schema Registry.                                                                                                                                             |schema_registry_url|
-|KPOPS_REST_PROXY_HOST    |             |False   |Address of the Kafka REST Proxy.                                                                                                                                            |kafka_rest_host    |
-|KPOPS_CONNECT_HOST       |             |False   |Address of Kafka Connect.                                                                                                                                                   |kafka_connect_host |
-|KPOPS_TIMEOUT            |          300|False   |The timeout in seconds that specifies when actions like deletion or deploy timeout.                                                                                         |timeout            |
-|KPOPS_RETAIN_CLEAN_JOBS  |False        |False   |Whether to retain clean up jobs in the cluster or uninstall the, after completion.                                                                                          |retain_clean_jobs  |
+|          Name           |    Default Value    |Required|                                                                                Description                                                                                 |  Setting name   |
+|-------------------------|---------------------|--------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------|
+|KPOPS_ENVIRONMENT        |                     |True    |The environment you want to generate and deploy the pipeline to. Suffix your environment files with this value (e.g. defaults_development.yaml for environment=development).|environment      |
+|KPOPS_KAFKA_BROKERS      |                     |True    |The comma separated Kafka brokers address.                                                                                                                                  |kafka_brokers    |
+|KPOPS_SCHEMA_REGISTRY_URL|http://localhost:8081|False   |Address of the Schema Registry.                                                                                                                                             |url              |
+|KPOPS_KAFKA_REST_URL     |http://localhost:8082|False   |Address of the Kafka REST Proxy.                                                                                                                                            |url              |
+|KPOPS_KAFKA_CONNECT_URL  |http://localhost:8083|False   |Address of Kafka Connect.                                                                                                                                                   |url              |
```

Comment:
hm these now all show as the same setting name since they're nested. doesn't make it particularly easy to understand imo...
it would be better to show the setting name as `schema_registry.url` etc.

Reply:
@sujuka99 Is it possible to do something like that?

Reply:
Should be, I'll fix it

Reply:
#366